### ISSUE TITLE
[Meta Gem] Mystical Skyfire Diamond does not proc 

### ISSUE THREAD

**Issue by cabapek** [ccshiro/cc-buglist#2016]
http://wowwiki.wikia.com/wiki/Mystical_Skyfire_Diamond

This is a Metagem which should give the caster a buff called Focus. This buff reduces the next spellcast time to half patch 2.0 or give 320 spell haste rating (20%) for 4 seconds.
It seems like the proc chance increased from patch 2.0 (elaboration on this : http://wowwiki.wikia.com/wiki/Mystical_Skyfire_Diamond).

Reproduction of the bug:
-Create a mage, level it up to 70 and get one head-slot item with a meta socket.
-Purchase Mystical Skyfire Diamond and other gems to make it work (# Blue gems > # Yellow gems). Seems to work (the tooltip isn't grey) when you have only one blue gem and no yellow gems.
-Cast frostbolts.

Expected result: After some frostbolts, you should get the Focus buff.

Result: Even after more than 50 frostbolts you don't get the Focus buff.

Sources of information:

http://www.wowhead.com/item=25893/mystical-skyfire-diamond#comments

http://wowwiki.wikia.com/wiki/Mystical_Skyfire_Diamond

**Comment by Jumpka** [comment]
Created a mage, leveled it up to 70, got s1 head with 10 spell pene socket (later tested with +8 spirit, +3 mp5 and +12 stam) and Mystical skyfire Diamond, casted few frostbolts, meta procced (got it like 4 times) so I'm not sure what happened in your tests.
Possible sources of errors:
1) not praying enough to the RNGesus and testing only for short period of time
2) your parents not telling you that you are color blind

The real question is, whether the proc chance is correct.

**Comment by Lephidiles** [comment]
probably had the haste proc trinket equipped and a buff stacking thing
Edit: or shared CD

**Comment by cabapek** [comment]
I did around 50 forstbolst last time and none if it worked, but I tried it again just now and I got procs constantly. I must have been doing something wrong but don't know why. I didn't had the 320 haste trinket for 6s on. But I tried now and with the trinket and both work ( sometimes the 4s pops, sometimes the 6s does). I'm new to this, so can anyone tell me what do I have to do with the thread now? Delete it?
